frontend/test/metabase/scenarios/admin/settings/sso.cy.spec.js

```diff
@@ -0,0 +1,22 @@
+import { restore } from "__support__/e2e/cypress";
+
+describe("scenarios > admin > settings > SSO", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it.skip("Google sign-in client ID should save on subsequent tries (metabase#15974)", () => {
+    cy.visit("/admin/settings/authentication/google");
+    cy.findByPlaceholderText("Your Google client ID").type("123");
+    saveSettings();
+
+    cy.findByDisplayValue("123").type("456");
+    saveSettings();
+  });
+});
+
+function saveSettings() {
+  cy.button("Save Changes").click();
+  cy.findByText("Saved");
+}
```

Comment:
I don't think this will work, since it's actually doing a page refresh on submit. That's the problem. It's a race-condition, so if the refresh happens too quickly, then it didn't have enough time to PUT before refreshing.
I think updating to the "new" Form is the only solution, and then it won't do a form-submit-refresh anymore.

Reply:
In the second screenshot I provided (sanity check) it seems that it's working.